code/modules/uplink/uplink_items.dm

```diff
@@ -1467,7 +1467,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
 	Insert into an Operating Console to enable the procedure."
 	item = /obj/item/disk/surgery/brainwashing
-	cost = 9
+	cost = 5
```

Comment:
This was denied on your other PR and shoehorned into this one. No.

Reply:
It was merged in the other PR, which was also not their PR

https://github.com/Skyrat-SS13/Skyrat-tg/pull/7860

Reply:
It's already merged in another PR